src/content/docs/concepts/security/capabilities.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: Capabilities
+---
+
+Tauri provides application and plugin developers with a capabilities system,
+to granually enable and constrain the core exposure to the application frontend running in the
+system WebView.
+
+Capabilities are a set of [permissions](/concepts/security/permissions) mapped
+to application windows and webviews by their respective label.
+Capabilities can affect multiple windows and webviews and these can be
+referenced in multiple capabilities.
+
+:::tip[Security Tip]
+
+Windows and WebViews which are part of more than one capability
+effectivly merge the security boundaries and permissions of all
+involved capabilities.
+
+:::
+
+Capability files are either defined as a JSON or a TOML file
+inside the `src-tauri/capabilities` directory.
+
+It is good practice to use individual files and only reference
+them by identifier in the `tauri.conf.json` but it also possible
+to defined them directly in the `capabilities` field.
+
+For a full reference of the configuration scheme please see the
+[references](/references/v2/config) section.
+
+The following example JSON defines a capability that enables default functionality
+for core plugins and the `window.setTitle` API.
+
+```json title="src-tauri/capabilities/main.json
+{
+	"$schema": "./schemas/desktop-schema.json",
+	"identifier": "main-capability",
+	"description": "Capability for the main window",
+	"windows": ["main"],
+	"permissions": [
+		"path:default",
+		"event:default",
+		"window:default",
+		"app:default",
+		"resources:default",
+		"menu:default",
+		"tray:default",
+		"window:allow-set-title"
+	]
+}
+```
+
+These snippets are part of the
+[Tauri configuration](/references/configuration-files#tauri-config) file.
+
+This is likely the most common configuration method,
+where the individual capabilities are inlined and only
+permissions are referenced by identifier.
+
+This requires well defined
+capability files in the `capabilities` directory.
+
+```json
+"capabilities":[ "my-capability", "main-capability"]
+```
+
+Inline capabilities can be mixed with pre-defined capabilities.
+
+
+```json
+"capabilities":[
+    {
+        "identifier": "my-capability",
+        "description": "My application capability used for all windows",
+        "windows": [ "*" ],
+        "permissions": [
+            "fs:default",
+            "allow-home-read-extended"
+           ]
+    },
+    "my-second-capability"
+]
+```
```

Comment:
```suggestion
```json title=src-tauri/tauri.conf.json
{
  "app": {
    "security": {
      "capabilities": [
	  {
	    "identifier": "my-capability",
	    "description": "My application capability used for all windows",
	    "windows": [ "*" ],
	    "permissions": [
	      "fs:default",
	      "allow-home-read-extended"
	     ]
	  },
	  "my-second-capability"
      ]
    }
  }
}
```
```